### PR TITLE
feat: add community odoo, an ERP and CRM system

### PR DESCRIPTION
**Pull request by alice39**
image

Here is a link to open the template: https://idx.google.com/new?template=https:%2F%2Fgithub.com%2Falice39%2Fcommunity-templates%2Ftree%2Fodoo%2Fodoo

### CODE REVIEW

**Comment by prakhar1989** [comment]
Thanks for the PR, @alice39 , can you share a screenshot or a screencast in the PR description?

**Comment by alice39** [comment]
@prakhar1989 done !

File: odoo/idx-template.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Odoo",
+  "description": "A suite of web based open source business apps",
+  "categories": [
+    "Web app",
+    "Backend"
+  ],
+  "icon": "",
```
Comment (prakhar1989) [inline odoo/idx-template.json:8]: worth setting this to `https://odoocdn.com/openerp_website/static/src/img/assets/png/odoo_logo_inverted.png` ?

**Comment by prakhar1989** [comment]
Worked perfectly! Nicely done!

**Review by prakhar1989 — CHANGES_REQUESTED** [review]

File: odoo/idx-template.json (hunk 1 above)
Comment (alice39) [reply odoo/idx-template.json:8]: Sure thing, lemme go for it

**Pull request merged**